themes/src/main/resources-community/theme/base/login/messages/messages_fr.properties

```
@@ -281,3 +281,9 @@ noCertificate=[Pas de certificat]
 
 pageNotFound=Page non trouv\u00e9e
 internalServerError=Une erreur interne du serveur s''est produite
+
+# Identity provider
+idp-email-verification-display-name=Vérification du courriel
```

I did not double check if unicode is really required but as the rest of translations are using it:
```suggestion
idp-email-verification-display-name=V\u00e9rification du courriel
```

---

done